OutOfSchool/OutOfSchool.WebApi/Services/WorkshopServicesCombiner.cs

```diff
@@ -171,13 +207,27 @@ public async Task<List<WorkshopCard>> GetByProviderId(Guid id)
             return workshopCards.ToList();
         }
 
-        // <inheritdoc/>
+        /// <inheritdoc/>
         public async Task<Guid> GetWorkshopProviderId(Guid workshopId) =>
             await workshopService.GetWorkshopProviderOwnerIdAsync(workshopId).ConfigureAwait(false);
 
-        private List<WorkshopCard> DtoModelsToWorkshopCards(IEnumerable<WorkshopDTO> source)
+        public async Task<IEnumerable<string>> GetNotificationsRecipientIds(NotificationAction action, Dictionary<string, string> additionalData, Guid objectId)
         {
-            return source.Select(currentElement => currentElement.ToCardDto()).ToList();
+            var recipientIds = new List<string>();
+
+            Expression<Func<Favorite, bool>> filter = x => x.WorkshopId == objectId;
+            var favoriteWorkshopUsersIds = favoriteRepository.Get(where: filter).Select(x => x.UserId).ToList();
```

Comment:
There is not need in the `filter` variable. Please change this to async call.
```suggestion
            var favoriteWorkshopUsersIds = await favoriteRepository.Get(where: x => x.WorkshopId == objectId)
                .Select(x => x.UserId)
                .ToListAsync()
                .ConfigureAwait(false);
```